src/org/usfirst/frc/team930/robot/controller/BBSController.java

```diff
@@ -0,0 +1,123 @@
+package org.usfirst.frc.team930.robot.controller;
+
+import edu.wpi.first.wpilibj.SpeedController;
+
+/**
+ * Concrete class for the implementation of the Bang Bang Shooter Controller,
+ * which will set the motor speed to max speed if the motor runs below the
+ * target RPM and will let it run freely otherwise (ie, speed of zero).
+ * <p>
+ * For best results, tune your acceleration values of the speed controller so
+ * that it doesn't jump around the set point - note, it will still control to
+ * your set points, but it will consistently overaccelerate in the calculation
+ * timeframe. As a reference, accel = 1 will produce ~3800 RPM
+ * <p>
+ * Based on the implementation on <a
+ * href="http://www.chiefdelphi.com/media/papers/2663">Chief Delphi</a>.
+ * 
+ * @see Controller.java
+ * @author Nick Janovetz
+ * @version 2016.02.03_3 Added finality to m_sc
+ */
+public class BBSController extends Controller {
+
+	private static int count = 0;
+
+	private ControllerSource m_source;
```

Comment:
Hungarian notation (the m's and s's before member and class variables) is kind of a blight on Java. Try and avoid it. It exists because there was a time when IDE's didn't make it apparent what kind of variable each was.